tests/pipeline/test_pipeline.py

```diff
@@ -458,8 +459,33 @@ def test_default_config(self, snapshot: SnapshotTest):
 
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
+    def test_env_vars_precedence_over_config(
+        self,
+        snapshot: SnapshotTest,
+    ):
+        os.environ["KPOPS_KAFKA_BROKERS"] = "env_broker"
```

Comment:
consider using `monkeypatch.setenv` in tests. not sure if `os.environ` has any downsides tbh